azure-pipelines.yml

```diff
@@ -23,3 +23,160 @@ jobs:
   parameters:
     name: WindowsPy27
     vmImage: vs2017-win2016
+
+- job: 'Checks_and_doc'
+  pool:
+    vmImage: ubuntu-16.04
+  timeoutInMinutes: 90
+  steps:
+  - script: |
+      # XXX not sure why setting $PATH here does not work
+      # echo '##vso[task.setvariable variable=PATH]$HOME/miniconda3/bin:$PATH'
```

Comment:
FWIW, you can `echo "##vso[task.prependpath]$HOME/miniconda3/bin"` here to make the path change durable for the job.